jstz_api/src/console.rs

```diff
@@ -135,15 +189,24 @@ fn formatter(data: &[JsValue], context: &mut Context<'_>) -> JsResult<String> {
     }
 }
 
-#[derive(Trace, Finalize)]
+#[derive(Finalize)]
 struct Console {
     groups: Vec<String>,
+    // TODO: Remove these once `Jstz` object is implemented
+    pub contract_address: PublicKeyHash,
+    pub operation_hash: Blake2b,
```

Comment:
I think both of these have dedicated types, namely `Address` and `OperationHash`